Action.c

```diff
@@ -569,48 +569,69 @@ static Htop_Reaction actionHelp(State* st) {
 
    attrset(CRT_colors[DEFAULT_COLOR]);
    line++;
-   mvaddstr(line++, 0, "CPU usage bar: ");
+   mvaddstr(line, 0, "CPU usage bar: ");
+
+   int bar_x = 0;
+
+#define addfirstbartext(attr, text) \
+   do { \
+      bar_x = getcurx(stdscr); \
+      addattrstr((attr), (text)); \
+   } while(0)
+#define addbartext(attr, text) \
```

Comment:
I think the macro name would be more readable if you name it in camelCase.
Otherwise I could mis-read it as "addBartExt"